browser/ui/webui/speedreader/speedreader_panel_ui.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/webui/speedreader/speedreader_panel_ui.h"
+
+#include <utility>
+
+#include "brave/browser/ui/brave_browser_window.h"
+#include "brave/browser/ui/webui/brave_webui_source.h"
+#include "brave/browser/ui/webui/speedreader/speedreader_panel_handler_impl.h"
+#include "brave/components/l10n/common/locale_util.h"
+#include "brave/components/speedreader/resources/panel/grit/brave_speedreader_panel_generated_map.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser_finder.h"
+#include "components/grit/brave_components_resources.h"
+#include "components/prefs/pref_service.h"
```

Comment:
not used? Pls check for unused includes through the PR

Reply:
all of them are used except pref service